react-backend/shared/models/cb-admin.js

```diff
@@ -0,0 +1,32 @@
+const then = new Date('2000-01-01T00:00:00.000Z');
+const now = new Date(Date.now());
+
+module.exports = {
+  getFeedback: async (dbConnection, { cbId, since = then, until = now }) => {
+    const getFeedbackQuery = `SELECT feedback_score, feedback_date FROM feedback WHERE cb_id = $1 AND feedback_date BETWEEN $2 AND $3`;
```

Comment:
Is there an endpoint where this query is exposed?

Reply:
Not yet.